src/engine/Server.cpp

```diff
@@ -16,6 +16,13 @@
 #include "./Server.h"
 #include "QueryPlanner.h"
 
+// _____________________________________________________________________________
+Server::~Server() {
+  if (_initialized) {
+    _serverSocket.close();
+  }
+}
+
```

Comment:
Probably never a problem if the Server is the only thing running in a problem but nevertheless always important to clean up open network stuff.